### PR TITLE
update OperationalInsights sdk version reference.

### PR DESCRIPTION
**Pull request by haitch**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@haidaocht__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (haitch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1050/

LGTM once the builds pass

File: src/ResourceManager/OperationalInsights/Commands.OperationalInsights.Test/packages.config [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.OperationalInsights" version="0.15.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.OperationalInsights" version="0.16.0-preview" targetFramework="net45" />
```
Comment (markcowl) [inline src/ResourceManager/OperationalInsights/Commands.OperationalInsights.Test/packages.config]: You are referring to older versions of Microsoft.IdentityModel.Clients.ActiveDirectory (2.28.0) and Microsoft.Rest.ClientRuntime.Azure.Authentication (2.2.6-preview) in this file - please update to 2.28.1 and 2.2.7-preview respectively, in your packages.config and csproj for both your cmdlets project and your test project

Comment (haitch) [inline src/ResourceManager/OperationalInsights/Commands.OperationalInsights.Test/packages.config]: fixed

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1059/

signing run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/551/

**Comment by markcowl** [comment]
@haitch You need to pull the latest from the dev branch into your pr - this is failing for assembly conflicts in Microsoft.IdentityModel.Clients.ActiveDirectory

**Comment by cormacpayne** [comment]
On demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1065/

**Comment by haitch** [comment]
trying to merge to azure/azure-powershell/release-2.0.0

**Pull request closed without merging**